.gitignore

```diff
@@ -3,4 +3,6 @@ test
 *.d
 depinst
```

Comment:
Note that we can probably remove `depinst` and `depsrc` since they are no longer used. Everything is contained in the libsnark folder